README.md

```diff
@@ -24,6 +24,19 @@ yo phovea
  * Yeoman can be too opinionated at times but is easily convinced not to be.
  * Feel free to [learn more about Yeoman](http://yeoman.io/).
 
+## Commands
+
+| Command                    | Description  |
+|----------------------------|--------------|
+| `yo phovea` or `yo phovea:app` | Initialize a new web application, library bundle, web library, server library, or service |
```

Comment:
I wouldn't add `yo phovea:app` it will be implicitly called. However, there are specific generators like `yo phovea:init-app` + there are hybrids: (web + server library), (application + server library)